index.js

```diff
@@ -0,0 +1,103 @@
+document.addEventListener("DOMContentLoaded", () =>{
+let startGame = document.querySelector("#startGame");
+let hit = document.querySelector("#hit")
+let computerCards = document.querySelector("#computerCards")
+let playerCards = document.querySelector("#playerCards");
+let gameScore = document.querySelector("gameScore");
+let playerScoreCard = 0; 
+let computerScoreCard = 0;
+let deckId = 'Not a Number Yet!';
+
+let obj = {
+    playerCards: 0,
+    computerCards: 0,
+}
+    
+    const shuffleDeck = async () =>{
+        try {
+            let { data: {deck_id} } = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1");
+            deckId = deck_id
+            console.log('deckId', deck_id);
+            
+        }catch(error) {
+            console.log("error");
+        }
+    }
+    
+    const changeValue = (value, score) => {
+        if(value === "QUEEN" || value === "KING" || value === "JACK") {
+            score += 10
+        } else if(value === "ACE") {
+            if(score < 11) {
+                score += 11
+            } else {
+                score += 1
+            } 
+        } else {
+            score += Number(value)
+        }
+        return score
+    }
+
+
+    const drawCard = async (num, div, score) => {
+        try{
+            let {data: {cards}} = await axios.get(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=${num}`);
+            // let cards = cards;
+            // debugger
+            console.log('cards', cards)
+            cards.forEach(card => {
+                console.log('card', card);
+                
+                let cardValue = card.value
+                let h2 = document.createElement("h2")
```

Comment:
you can create this score h2 outside this function and then only update that score when you use this function.